Server/src/Server.Api/Controllers/LibrariesController.cs

```diff
@@ -40,4 +41,27 @@ public async Task<ActionResult> Post(Guid id)
         await Mediator.Send(new RemoveLibraryCommand(id));
         return NoContent();
     }
+
+    [HttpPost("{id:guid}/books")]
+    public async Task<ActionResult> AddBook(Guid id, AddBookToLibraryCommand newBookInLibrary)
+    {
+        await Mediator.Send(newBookInLibrary with
+        {
+            LibraryId = id
+        });
+
+        return Ok();
+    }
+
+    [HttpGet("{id:guid}/books")]
+    public async Task<ActionResult<IEnumerable<BookInLibraryViewModel>>> GetBooks(Guid id)
+    {
+        return Ok(await Mediator.Send(new GetBooksInLibraryQuery(id)));
+    }
+
+    [HttpGet("{id:guid}/booksToAdd")]
```

Comment:
łączone nazwy w ścieżce chyba ładniej będzie robić kebab casem

Reply:
i zastanawiam się czy inna nazwa typu `id/books/not-added` by nie była lepsza bo trochę nie zczaiłem o co chodzi z tym query

Reply:
racja, poprawię